### DISCUSSION TITLE
萨缪尔森说 | /cn/2005/05/samuelson/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1569] [Comments]
“大多数伟大的经济学家在他们的自传中都告诉我们，他们之所以成为经济学家，是因为他们有志于改良社会。”
[&hellip;] 今天刚刚看到这句话，要记下来。

https://yihui.org/cn/2005/05/samuelson/

**Comment by think-crow** [comment]
老歪大哥好哈！没曾想看你文章还有宁心静神功能，带着浮躁的心来，怀着平静的心去；哈哈哈哈！

今天先看到这里，祝您老生活愉快，工作顺利，文章多多！

**Reply by yihui** [reply]
谢谢！
